requirements.txt

```diff
@@ -2,9 +2,9 @@ bayescd>=0.4
 changepy>=0.3.1
 hyperopt>=0.1.2
 numpy>=1.17.5, <=1.22.4
-pandas>=0.25.3
+pandas>=0.25.3, <=2.0.3
 pykalman>=0.9.5
 scipy>=1.6.0
-statsmodels>=0.13.0
```

Comment:
Need to add upper bounds due to this serialization issue with pandas and higher statsmodels versions. Will open a separate issues to look into these.